code/datums/shuttles.dm

```diff
@@ -161,6 +161,13 @@
 	description = "A small, but feature complete shuttle. It boasts a card table to keep crew members occupied on the long flight home."
 	credit_cost = 1000
 
+/datum/map_template/shuttle/emergency/cere
+	suffix = "cere"
+	name = "Cere Station Emergency Shuttle"
+	description = "The large, beefed-up version of the box-standard shuttle. Includes an expanded brig, fully stocked medbay, enhanced cargo storage with mech chargers, \
+		an engine room stocked with various supplies, and a crew capacity of 80+ to top it all off. Live large, live Cere."
+	credit_cost = INFINITY
```

Comment:
One last thing and I'll merge it! This is redundant, it's already set in the `map_template/shuttle` base ([here](https://github.com/tgstation/tgstation/blob/master/code/datums/shuttles.dm#L12)) !